app/src/main/java/com/sdp13epfl2021/projmag/adapter/ProjectAdapter.kt

```diff
@@ -140,6 +141,24 @@ class ProjectAdapter(private val context: Context, private val utils: Utils, pri
         }
     }
 
+
+    fun tagAndSectionsChipViewSetup(project: ImmutableProject, holder: ProjectViewHolder){
+        // add the tags to the project
+        for (tag in project.tags) {
+            val chipView: Chip = Chip(context)
+            chipView.text = tag
+            chipView.chipBackgroundColor = ColorStateList.valueOf(ContextCompat.getColor(context, R.color.light_green))
+            holder.chipGroupView.addView(chipView)
+        }
+
+        for(section in project.allowedSections){
+            val chipView: Chip = Chip(context)
+            chipView.text = section
+            chipView.chipBackgroundColor = ColorStateList.valueOf(ContextCompat.getColor(context, R.color.teal_700))
+            holder.chipGroupView.addView(chipView)
+        }
```

Comment:
The code seems a bit dupplicate, maybe adding a function like that would avoid it.
private fun addChips(textsList: List<String>, color: ...) {...}